src/assert.libsonnet

```diff
@@ -229,4 +229,575 @@
     name: name,
     value: value,
   }),
+
+  /*
+  hostOnly and session are not an attribute 
+  in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie 
+  or https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-12
+
+  GET https://httpbin.akam.cloud/response-headers?Set-Cookie=silver%3D
+  yields 
+  - Set-Cookie: silver=
+  - JSON.stringify(cookie): {"name":"silver","expires":null,"maxAge":0,"domain":"httpbin.akam.cloud","path":"/","secure":false,"httpOnly":false,"hostOnly":true}
+  */
+  assertCookieAttributeExists(title,name,attribute):: |||
+    (function (params) {
+      let attributeValue;
+      // Becomes true if cookie has been set in the cookie jar and (attribute always exists even its value is empty (cookie value), attribute is provided or attribute has a default value)
+      let attributeFound=false;
+      const cookie=pm.cookies.one(params.name);
+      //console.log("JSON.stringify(pm.cookies.toObject(): <" + JSON.stringify(pm.cookies.toObject()) + ">");
+      if (cookie){
+        switch(params.attribute) {
+          // Cookie value
+          case "":
```

Comment:
I thougth that it would be convenient to make asserCookieAttrtibute functions able to handle cookie value too but sure, I can remove this.

I used empty string because it's the only value i could think of that can't be the name of some unknown/future/custom/non compliant attribute. I could replace this by a string that includes cookie-value and some characters forbidden  (preferably impossible) in cookie attribute names.

When cookie exists and its value is empty, assertCookieAttributeEquals and asserCookieAttributeMatches match with empty string. I prefer this over assertCookieEquals and asserCookieMatches, which match with undefined, which is same value as when cookie doesn't exist.

I would suggest to update assertCookieAttributeEquals and asserCookieAttributeMatches to match with empty string when cookie exists and its value in the pm objects is undefined.